### DISCUSSION TITLE
channel is open but gives closed channel error

### DISCUSSION THREAD

**Discussion by Ayejang** [ProjectOpenSea/seaport#1323] [Q&A] [answered]
I recently deployed conduit using Conudit Contoller in Amoy testnet.
I update the channel status to true, which I was using in mumbai. (the zone is `0x0000000000000000000000000000000000000092`) 
but when I try to buy selling nft in my marketplace, it gives me error like this.

> ERROR:  {"code": 3, "data": "0x93daadf200000000000000000000000000000000006c3852cbef3e08e8df289169ede581", "message": "execution reverted"}

I found out that the error data says the channel is closed. ([link](https://www.codeslaw.app/contracts/ethereum/0x1e0049783f008a0085193e00003d00cd54003c71?file=contracts%2Fconduit%2Flib%2FConduitConstants.sol&start=6))
but I've already open the channel, as you can see in the [link](https://www.oklink.com/ko/amoy/address/0x2ff0870e016d6287efcfb43e88323850a29ee960/event)

is there anything I am missing?

(+) I am using seaport 1.1

**Comment by ryanio** [comment, marked as answer]
0x2fF0870E016d6287eFcfb43e88323850a29ee960 is the conduit you created, so you have to use that for the conduitKey. Also you have to manually add Seaport as an open channel to it, it's not open by default.

**Reply by Ayejang** [reply]
after opening channel of seaport, the problem has solved. thank you so much!!
